modules/ding_user_frontend/ding_user_frontend.module

```diff
@@ -329,6 +329,17 @@ function ding_user_frontend_build_menu($path = NULL) {
         }
       }
     }
+
+    // Add logout to menu.
+    $logout_item = array();
+    $logout_item = $menu_item_defaults;
+    $logout_item['#title'] = t('Log out');
+    $logout_item['#href'] = '/user/logout';
```

Comment:
It's a render array, using the `theme_menu_link`, so it's passed through the `l()` function later on in that theme.